promscale/installation/index.md

```diff
@@ -10,6 +10,16 @@ from your applications and infrastructure. It is expected that you use
 instrumentation set up, you can use Promscale to ingest the metric and
 trace data.
 
+<highlight type="note">
+The PostgreSQL `search_path` variable allows you to control in what order
+schemas are searched and which objects (tables, views, functions, etc.) do not
+require schema qualification to use. When you install Promscale, the Promscale
+extension modified the `search_path` of the database that it's connected to,
+adding its public schemas to the search path. Promscale's public schemas are:
+`ps_tag`, `prom_api`, `prom_metric`, `ps_trace`. This makes querying Promscale
+data easier.
```

Comment:
The PostgreSQL `search_path` variable determines in what order schemas
are searched and which objects such as tables, views, functions, and others
do not require schema qualification. When you install Promscale, the Promscale
extension modifies the `search_path` of the database that it is connected to
and adds its public schemas to the search path. This makes querying Promscale
data easier. The public schemas that Promscale adds are:
`ps_tag`, `prom_api`, `prom_metric`, and `ps_trace`.

Reply:
done